### PR TITLE
Add batch 5

### PR DESCRIPTION
**Pull request by Yullia**

### CODE REVIEW

File: meta/48fd3902.csv [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+86809,eaeb2b79,GitHub,48fd3902,data/48fd3902/test/eaeb2b79.java,333,333,F,F,78,91,F,F,,,,,2.90,13,F,F,F,Password
+86810,eaeb2b79,GitHub,48fd3902,data/48fd3902/test/eaeb2b79.java,761,761,F,F,90,103,F,F,,,,,2.90,13,F,F,F,Password
+86811,eaeb2b79,GitHub,48fd3902,data/48fd3902/test/eaeb2b79.java,817,817,F,F,62,75,F,F,,,,,2.90,13,F,F,F,Password
+86812,eaeb2b79,GitHub,48fd3902,data/48fd3902/test/eaeb2b79.java,826,826,F,F,57,70,F,F,,,,,2.90,13,F,F,F,Password
+86813,f1536ec6,GitHub,48fd3902,data/48fd3902/test/f1536ec6.kt,83,83,F,F,31,42,F,F,,,,,2.96,11,F,F,F,Certificate
+86814,f7570570,GitHub,48fd3902,data/48fd3902/test/f7570570.java,46,46,F,F,16,29,F,F,,,,,2.90,13,F,F,F,Password
+86815,f7570570,GitHub,48fd3902,data/48fd3902/test/f7570570.java,70,70,F,F,29,50,F,F,,,,,3.37,21,F,F,F,Password
+86816,f7570570,GitHub,48fd3902,data/48fd3902/test/f7570570.java,88,88,F,F,28,49,F,F,,,,,3.37,21,F,F,F,Password
+86817,ff2a9656,GitHub,48fd3902,data/48fd3902/test/ff2a9656.java,75,75,F,F,21,37,F,F,,,,,3.25,16,F,F,F,Password
+109447,28858c37,GitHub,48fd3902,data/48fd3902/other/28858c37.md,80,86,F,F,0,81,F,F,,,,,5.5361776,,F,F,F,Certificate
```
Comment (babenek) [inline meta/48fd3902.csv:178]:
```suggestion
109447,28858c37,GitHub,48fd3902,data/48fd3902/other/28858c37.md,80,86,F,F,0,81,F,F,,,,,5.5361776,,F,F,F,Private Key
```

File: meta/48fd3902.csv [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+86493,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,403,429,T,F,1,32,F,F,,,,,2.53,31,F,F,F,PEM Private Key
+86494,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,206,232,T,F,1,65,F,F,,,,,5.08,64,F,F,F,PEM Private Key
+86519,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,236,262,T,F,1,28,F,F,,,,,2.39,27,F,F,F,PEM Private Key
+86546,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,266,270,F,F,41,71,F,F,,,,,2.45,30,F,F,F,PEM Private Key
+86550,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,323,349,F,F,44,75,F,F,,,,,2.53,31,F,F,F,PEM Private Key
+86576,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,351,377,F,F,50,77,F,F,,,,,2.39,27,F,F,F,PEM Private Key
+86628,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,458,508,T,F,50,81,F,F,,,,,2.53,31,F,F,F,PEM Private Key
+86678,238cc5a0,GitHub,48fd3902,data/48fd3902/other/238cc5a0.md,53,53,F,F,77,97,F,F,,,,,3.22,21,F,F,F,Password
+86679,238cc5a0,GitHub,48fd3902,data/48fd3902/other/238cc5a0.md,62,62,F,F,77,97,F,F,,,,,3.22,21,F,F,F,Password
+86680,28858c37,GitHub,48fd3902,data/48fd3902/other/28858c37.md,97,97,T,F,13,29,F,F,,,,,2.30,13,F,F,F,Password
```
Comment (babenek) [inline meta/48fd3902.csv:40]:
```suggestion
86680,28858c37,GitHub,48fd3902,data/48fd3902/other/28858c37.md,97,97,T,F,12,29,F,F,,,,,2.30,13,F,F,F,Password
```

File: meta/48fd3902.csv [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+86811,eaeb2b79,GitHub,48fd3902,data/48fd3902/test/eaeb2b79.java,817,817,F,F,62,75,F,F,,,,,2.90,13,F,F,F,Password
+86812,eaeb2b79,GitHub,48fd3902,data/48fd3902/test/eaeb2b79.java,826,826,F,F,57,70,F,F,,,,,2.90,13,F,F,F,Password
+86813,f1536ec6,GitHub,48fd3902,data/48fd3902/test/f1536ec6.kt,83,83,F,F,31,42,F,F,,,,,2.96,11,F,F,F,Certificate
+86814,f7570570,GitHub,48fd3902,data/48fd3902/test/f7570570.java,46,46,F,F,16,29,F,F,,,,,2.90,13,F,F,F,Password
+86815,f7570570,GitHub,48fd3902,data/48fd3902/test/f7570570.java,70,70,F,F,29,50,F,F,,,,,3.37,21,F,F,F,Password
+86816,f7570570,GitHub,48fd3902,data/48fd3902/test/f7570570.java,88,88,F,F,28,49,F,F,,,,,3.37,21,F,F,F,Password
+86817,ff2a9656,GitHub,48fd3902,data/48fd3902/test/ff2a9656.java,75,75,F,F,21,37,F,F,,,,,3.25,16,F,F,F,Password
+109447,28858c37,GitHub,48fd3902,data/48fd3902/other/28858c37.md,80,86,F,F,0,81,F,F,,,,,5.5361776,,F,F,F,Certificate
+109449,95ce83e8,GitHub,48fd3902,data/48fd3902/other/95ce83e8.md,54,54,T,F,13,53,F,F,,,,,4.603056,,F,F,F,Token
+109450,95ce83e8,GitHub,48fd3902,data/48fd3902/other/95ce83e8.md,70,155,F,F,16,53,F,F,,,,,5.0281944,,F,F,F,Certificate
```
Comment (babenek) [inline meta/48fd3902.csv:180]:
```suggestion
109450,95ce83e8,GitHub,48fd3902,data/48fd3902/other/95ce83e8.md,70,79,F,F,16,16,F,F,,,,,5.0281944,,F,F,F,Generic Secret
```

File: meta/48fd3902.csv [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+86550,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,323,349,F,F,44,75,F,F,,,,,2.53,31,F,F,F,PEM Private Key
+86576,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,351,377,F,F,50,77,F,F,,,,,2.39,27,F,F,F,PEM Private Key
+86628,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,458,508,T,F,50,81,F,F,,,,,2.53,31,F,F,F,PEM Private Key
+86678,238cc5a0,GitHub,48fd3902,data/48fd3902/other/238cc5a0.md,53,53,F,F,77,97,F,F,,,,,3.22,21,F,F,F,Password
+86679,238cc5a0,GitHub,48fd3902,data/48fd3902/other/238cc5a0.md,62,62,F,F,77,97,F,F,,,,,3.22,21,F,F,F,Password
+86680,28858c37,GitHub,48fd3902,data/48fd3902/other/28858c37.md,97,97,T,F,13,29,F,F,,,,,2.30,13,F,F,F,Password
+86681,85c18102,GitHub,48fd3902,data/48fd3902/other/85c18102.md,42,42,F,F,19,30,F,F,,,,,2.33,11,F,F,F,Password
+86682,85c18102,GitHub,48fd3902,data/48fd3902/other/85c18102.md,43,43,F,F,24,38,F,F,,,,,2.45,16,F,F,F,Password
+86683,85c18102,GitHub,48fd3902,data/48fd3902/other/85c18102.md,116,116,F,F,12,28,F,F,,,,,2.33,11,F,F,F,Password
+86684,95ce83e8,GitHub,48fd3902,data/48fd3902/other/95ce83e8.md,204,204,F,F,22,31,F,F,,,,,2.33,11,F,F,F,Password
```
Comment (babenek) [inline meta/48fd3902.csv:44]:
```suggestion
86684,95ce83e8,GitHub,48fd3902,data/48fd3902/other/95ce83e8.md,204,204,F,F,,,F,F,,,,,2.33,11,F,F,F,Password
```

File: meta/48fd3902.csv [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+86546,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,266,270,F,F,41,71,F,F,,,,,2.45,30,F,F,F,PEM Private Key
+86550,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,323,349,F,F,44,75,F,F,,,,,2.53,31,F,F,F,PEM Private Key
+86576,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,351,377,F,F,50,77,F,F,,,,,2.39,27,F,F,F,PEM Private Key
+86628,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,458,508,T,F,50,81,F,F,,,,,2.53,31,F,F,F,PEM Private Key
+86678,238cc5a0,GitHub,48fd3902,data/48fd3902/other/238cc5a0.md,53,53,F,F,77,97,F,F,,,,,3.22,21,F,F,F,Password
+86679,238cc5a0,GitHub,48fd3902,data/48fd3902/other/238cc5a0.md,62,62,F,F,77,97,F,F,,,,,3.22,21,F,F,F,Password
+86680,28858c37,GitHub,48fd3902,data/48fd3902/other/28858c37.md,97,97,T,F,13,29,F,F,,,,,2.30,13,F,F,F,Password
+86681,85c18102,GitHub,48fd3902,data/48fd3902/other/85c18102.md,42,42,F,F,19,30,F,F,,,,,2.33,11,F,F,F,Password
+86682,85c18102,GitHub,48fd3902,data/48fd3902/other/85c18102.md,43,43,F,F,24,38,F,F,,,,,2.45,16,F,F,F,Password
+86683,85c18102,GitHub,48fd3902,data/48fd3902/other/85c18102.md,116,116,F,F,12,28,F,F,,,,,2.33,11,F,F,F,Password
```
Comment (babenek) [inline meta/48fd3902.csv:43]:
```suggestion
86681,85c18102,GitHub,48fd3902,data/48fd3902/other/85c18102.md,42,42,F,F,,,F,F,,,,,2.33,11,F,F,F,Password
86682,85c18102,GitHub,48fd3902,data/48fd3902/other/85c18102.md,43,43,F,F,,,F,F,,,,,2.45,16,F,F,F,Password
86683,85c18102,GitHub,48fd3902,data/48fd3902/other/85c18102.md,116,116,F,F,,,F,F,,,,,2.33,11,F,F,F,Password
```

File: meta/48fd3902.csv [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+86815,f7570570,GitHub,48fd3902,data/48fd3902/test/f7570570.java,70,70,F,F,29,50,F,F,,,,,3.37,21,F,F,F,Password
+86816,f7570570,GitHub,48fd3902,data/48fd3902/test/f7570570.java,88,88,F,F,28,49,F,F,,,,,3.37,21,F,F,F,Password
+86817,ff2a9656,GitHub,48fd3902,data/48fd3902/test/ff2a9656.java,75,75,F,F,21,37,F,F,,,,,3.25,16,F,F,F,Password
+109447,28858c37,GitHub,48fd3902,data/48fd3902/other/28858c37.md,80,86,F,F,0,81,F,F,,,,,5.5361776,,F,F,F,Certificate
+109449,95ce83e8,GitHub,48fd3902,data/48fd3902/other/95ce83e8.md,54,54,T,F,13,53,F,F,,,,,4.603056,,F,F,F,Token
+109450,95ce83e8,GitHub,48fd3902,data/48fd3902/other/95ce83e8.md,70,155,F,F,16,53,F,F,,,,,5.0281944,,F,F,F,Certificate
+109452,1be4753a,GitHub,48fd3902,data/48fd3902/test/1be4753a.java,42,42,F,F,12,32,F,F,,,,,4.3685226,,F,F,F,Password
+109456,379ffa27,GitHub,48fd3902,data/48fd3902/test/379ffa27.key,1,54,T,F,0,0,F,F,,,,,6.047873,,F,F,F,Certificate
+109457,44b1cbf6,GitHub,48fd3902,data/48fd3902/test/44b1cbf6.java,167,201,F,F,44,0,F,F,,,,,5.2023597,,F,F,F,Certificate
+109458,41abbdc5,GitHub,48fd3902,data/48fd3902/test/41abbdc5.java,100,100,F,F,25,34,F,F,,,,,3.0957952,,F,F,F,Password
```
Comment (babenek) [inline meta/48fd3902.csv:184]:
```suggestion
109458,41abbdc5,GitHub,48fd3902,data/48fd3902/test/41abbdc5.java,100,100,T,F,25,34,F,F,,,,,3.0957952,,F,F,F,Password
```

File: meta/48fd3902.csv [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+86814,f7570570,GitHub,48fd3902,data/48fd3902/test/f7570570.java,46,46,F,F,16,29,F,F,,,,,2.90,13,F,F,F,Password
+86815,f7570570,GitHub,48fd3902,data/48fd3902/test/f7570570.java,70,70,F,F,29,50,F,F,,,,,3.37,21,F,F,F,Password
+86816,f7570570,GitHub,48fd3902,data/48fd3902/test/f7570570.java,88,88,F,F,28,49,F,F,,,,,3.37,21,F,F,F,Password
+86817,ff2a9656,GitHub,48fd3902,data/48fd3902/test/ff2a9656.java,75,75,F,F,21,37,F,F,,,,,3.25,16,F,F,F,Password
+109447,28858c37,GitHub,48fd3902,data/48fd3902/other/28858c37.md,80,86,F,F,0,81,F,F,,,,,5.5361776,,F,F,F,Certificate
+109449,95ce83e8,GitHub,48fd3902,data/48fd3902/other/95ce83e8.md,54,54,T,F,13,53,F,F,,,,,4.603056,,F,F,F,Token
+109450,95ce83e8,GitHub,48fd3902,data/48fd3902/other/95ce83e8.md,70,155,F,F,16,53,F,F,,,,,5.0281944,,F,F,F,Certificate
+109452,1be4753a,GitHub,48fd3902,data/48fd3902/test/1be4753a.java,42,42,F,F,12,32,F,F,,,,,4.3685226,,F,F,F,Password
+109456,379ffa27,GitHub,48fd3902,data/48fd3902/test/379ffa27.key,1,54,T,F,0,0,F,F,,,,,6.047873,,F,F,F,Certificate
+109457,44b1cbf6,GitHub,48fd3902,data/48fd3902/test/44b1cbf6.java,167,201,F,F,44,0,F,F,,,,,5.2023597,,F,F,F,Certificate
```
Comment (babenek) [inline meta/48fd3902.csv:183]:
```suggestion
```

File: meta/48fd3902.csv [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+109458,41abbdc5,GitHub,48fd3902,data/48fd3902/test/41abbdc5.java,100,100,F,F,25,34,F,F,,,,,3.0957952,,F,F,F,Password
+109461,6e09cb2b,GitHub,48fd3902,data/48fd3902/test/6e09cb2b.java,24,24,T,F,29,43,F,F,,,,,3.5,,F,F,F,Password
+109462,710dfdfa,GitHub,48fd3902,data/48fd3902/test/710dfdfa.java,33,34,F,F,28,44,F,F,,,,,4.56337,,F,F,F,Certificate
```
Comment (babenek) [inline meta/48fd3902.csv:186]:
```suggestion
```

File: meta/48fd3902.csv [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+86191,28858c37,GitHub,48fd3902,data/48fd3902/other/28858c37.md,69,69,T,F,10,40,F,F,,,,,4.24,30,F,F,F,Password
+86193,dd2c0eb6,GitHub,48fd3902,data/48fd3902/other/dd2c0eb6,43,43,T,F,21,33,F,F,,,,,2.36,12,F,F,F,IPv4
+86195,8601ed21,GitHub,48fd3902,data/48fd3902/src/8601ed21.yml,7,7,T,F,13,1489,F,F,,,,,5.55,1476,F,F,F,Certificate
+86196,8601ed21,GitHub,48fd3902,data/48fd3902/src/8601ed21.yml,15,15,T,F,13,2285,F,F,,,,,5.62,2272,F,F,F,BASE64 encoded PEM Private Key
+86198,d10a5de0,GitHub,48fd3902,data/48fd3902/src/d10a5de0.yaml,7,7,T,F,18,32,F,F,,,,,2.37,14,F,F,F,IPv4
+86199,d10a5de0,GitHub,48fd3902,data/48fd3902/src/d10a5de0.yaml,8,8,T,F,17,29,F,F,,,,,2.36,12,F,F,F,IPv4
+86200,e33c22c3,GitHub,48fd3902,data/48fd3902/src/e33c22c3.yml,35,61,T,F,0,31,F,F,,,,,2.53,31,F,F,F,PEM Private Key
+86226,328e2512,GitHub,48fd3902,data/48fd3902/test/328e2512.groovy,8,8,T,F,32,946,F,F,,,,,5.71,914,F,F,F,Azure Access Token
+86227,4de5dba6,GitHub,48fd3902,data/48fd3902/test/4de5dba6.java,205,205,T,F,45,75,F,F,,,,,4.15,30,F,F,F,Password
+86229,73bf4bdd,GitHub,48fd3902,data/48fd3902/test/73bf4bdd.java,463,513,F,F,38,69,F,F,,,,,2.53,31,F,F,F,PEM Private Key
```
Comment (babenek) [inline meta/48fd3902.csv:11]:
```suggestion
86229,73bf4bdd,GitHub,48fd3902,data/48fd3902/test/73bf4bdd.java,463,513,T,F,38,69,F,F,,,,,2.53,31,F,F,F,PEM Private Key
```

File: meta/48fd3902.csv [hunk 10]
```diff
@@ -0,0 +1,193 @@
...
+86279,73bf4bdd,GitHub,48fd3902,data/48fd3902/test/73bf4bdd.java,518,544,F,F,15,46,F,F,,,,,2.53,31,F,F,F,PEM Private Key
```
Comment (babenek) [inline meta/48fd3902.csv:12]:
```suggestion
86279,73bf4bdd,GitHub,48fd3902,data/48fd3902/test/73bf4bdd.java,518,544,T,F,15,46,F,F,,,,,2.53,31,F,F,F,PEM Private Key
```

File: meta/48fd3902.csv [hunk 11]
```diff
@@ -0,0 +1,193 @@
...
+86310,8605db08,GitHub,48fd3902,data/48fd3902/test/8605db08.kt,46,46,T,F,29,1106,F,F,,,,,5.70,1077,F,F,F,Key
+86311,8605db08,GitHub,48fd3902,data/48fd3902/test/8605db08.kt,52,52,T,F,30,1205,F,F,,,,,5.63,1175,F,F,F,JSON Web Token
+86312,8605db08,GitHub,48fd3902,data/48fd3902/test/8605db08.kt,58,58,T,F,43,1051,F,F,,,,,5.71,1008,F,F,F,JSON Web Token
+86313,8605db08,GitHub,48fd3902,data/48fd3902/test/8605db08.kt,60,60,T,F,40,1082,F,F,,,,,5.70,1042,F,F,F,Azure Access Token
+86314,8605db08,GitHub,48fd3902,data/48fd3902/test/8605db08.kt,62,62,T,F,37,1054,F,F,,,,,5.68,1017,F,F,F,Azure Access Token
+86315,8605db08,GitHub,48fd3902,data/48fd3902/test/8605db08.kt,64,64,T,F,38,1067,F,F,,,,,5.63,1029,F,F,F,Azure Access Token
+86316,8605db08,GitHub,48fd3902,data/48fd3902/test/8605db08.kt,66,66,T,F,38,1118,F,F,,,,,5.67,1080,F,F,F,Azure Access Token
+86317,8605db08,GitHub,48fd3902,data/48fd3902/test/8605db08.kt,68,68,T,F,37,1097,F,F,,,,,5.66,1060,F,F,F,Token
+86318,8605db08,GitHub,48fd3902,data/48fd3902/test/8605db08.kt,70,70,T,F,35,1115,F,F,,,,,5.70,1080,F,F,F,Token
+86319,87e253cc,GitHub,48fd3902,data/48fd3902/test/87e253cc.java,81,131,F,F,5,36,F,F,,,,,2.53,31,F,F,F,PEM Private Key
```
Comment (babenek) [inline meta/48fd3902.csv:27]:
```suggestion
86319,87e253cc,GitHub,48fd3902,data/48fd3902/test/87e253cc.java,81,131,T,F,5,36,F,F,,,,,2.53,31,F,F,F,PEM Private Key
```

File: meta/48fd3902.csv [hunk 12]
```diff
@@ -0,0 +1,193 @@
...
+86369,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,87,137,F,F,1,32,F,F,,,,,2.53,31,F,F,F,PEM Private Key
+86370,c709b566,GitHub,48fd3902,data/48fd3902/test/c709b566.kt,442,492,F,F,1,65,F,F,,,,,5.12,64,F,F,F,PEM Private Key
+86419,c709b566,GitHub,48fd3902,data/48fd3902/test/c709b566.kt,496,522,F,F,1,65,F,F,,,,,5.05,64,F,F,F,PEM Private Key
```
Comment (babenek) [inline meta/48fd3902.csv:30]:
```suggestion
86419,c709b566,GitHub,48fd3902,data/48fd3902/test/c709b566.kt,496,522,T,F,1,65,F,F,,,,,5.05,64,F,F,F,PEM Private Key
```

File: meta/48fd3902.csv [hunk 13]
```diff
@@ -0,0 +1,193 @@
...
+109456,379ffa27,GitHub,48fd3902,data/48fd3902/test/379ffa27.key,1,54,T,F,0,0,F,F,,,,,6.047873,,F,F,F,Certificate
+109457,44b1cbf6,GitHub,48fd3902,data/48fd3902/test/44b1cbf6.java,167,201,F,F,44,0,F,F,,,,,5.2023597,,F,F,F,Certificate
+109458,41abbdc5,GitHub,48fd3902,data/48fd3902/test/41abbdc5.java,100,100,F,F,25,34,F,F,,,,,3.0957952,,F,F,F,Password
+109461,6e09cb2b,GitHub,48fd3902,data/48fd3902/test/6e09cb2b.java,24,24,T,F,29,43,F,F,,,,,3.5,,F,F,F,Password
+109462,710dfdfa,GitHub,48fd3902,data/48fd3902/test/710dfdfa.java,33,34,F,F,28,44,F,F,,,,,4.56337,,F,F,F,Certificate
+109477,9417bc40,GitHub,48fd3902,data/48fd3902/test/9417bc40.java,94,94,F,F,30,43,F,F,,,,,3.5068905,,F,F,F,Password
+109480,c6175528,GitHub,48fd3902,data/48fd3902/test/c6175528.kt,215,241,T,F,0,71,F,F,,,,,5.6877875,,F,F,F,Certificate
+109481,c6175528,GitHub,48fd3902,data/48fd3902/test/c6175528.kt,105,105,F,F,13,32,F,F,,,,,3.6304126,,F,F,F,Password
+109482,c9c69cbb,GitHub,48fd3902,data/48fd3902/test/c9c69cbb.kt,39,39,F,F,32,78,F,F,,,,,4.214022,,F,F,F,Token
+109484,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,9,87,F,F,54,78,F,F,,,,,5.5997286,,F,F,F,Certificate
```
Comment (babenek) [inline meta/48fd3902.csv:191]:
```suggestion
109484,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,9,35,T,F,54,78,F,F,,,,,5.5997286,,F,F,F,Certificate
```

File: meta/48fd3902.csv [hunk 14]
```diff
@@ -0,0 +1,193 @@
...
+86317,8605db08,GitHub,48fd3902,data/48fd3902/test/8605db08.kt,68,68,T,F,37,1097,F,F,,,,,5.66,1060,F,F,F,Token
+86318,8605db08,GitHub,48fd3902,data/48fd3902/test/8605db08.kt,70,70,T,F,35,1115,F,F,,,,,5.70,1080,F,F,F,Token
+86319,87e253cc,GitHub,48fd3902,data/48fd3902/test/87e253cc.java,81,131,F,F,5,36,F,F,,,,,2.53,31,F,F,F,PEM Private Key
+86369,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,87,137,F,F,1,32,F,F,,,,,2.53,31,F,F,F,PEM Private Key
+86370,c709b566,GitHub,48fd3902,data/48fd3902/test/c709b566.kt,442,492,F,F,1,65,F,F,,,,,5.12,64,F,F,F,PEM Private Key
+86419,c709b566,GitHub,48fd3902,data/48fd3902/test/c709b566.kt,496,522,F,F,1,65,F,F,,,,,5.05,64,F,F,F,PEM Private Key
+86493,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,403,429,T,F,1,32,F,F,,,,,2.53,31,F,F,F,PEM Private Key
+86494,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,206,232,T,F,1,65,F,F,,,,,5.08,64,F,F,F,PEM Private Key
+86519,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,236,262,T,F,1,28,F,F,,,,,2.39,27,F,F,F,PEM Private Key
+86546,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,266,270,F,F,41,71,F,F,,,,,2.45,30,F,F,F,PEM Private Key
```
Comment (babenek) [inline meta/48fd3902.csv:34]:
```suggestion
86546,e9b76075,GitHub,48fd3902,data/48fd3902/test/e9b76075.kt,266,270,T,F,41,71,F,F,,,,,2.45,30,F,F,F,PEM Private Key
```

**Review by babenek — COMMENTED** [review]
Corrections

**Review by babenek — APPROVED** [review]
Approved

**Review by xDizzix — DISMISSED** [review]

**Review by babenek — APPROVED** [review]
Approved

**Review by iuriimet — APPROVED** [review]

**Pull request merged**